mlflow/store/tracking/abstract_store.py

```diff
@@ -294,12 +295,64 @@ def delete_traces(
             experiment_id: ID of the associated experiment.
             max_timestamp_millis: The maximum timestamp in milliseconds since the UNIX epoch for
                 deleting traces. Traces older than or equal to this timestamp will be deleted.
-            max_traces: The maximum number of traces to delete.
+            max_traces: The maximum number of traces to delete. If max_traces is specified, and
+                it is less than the number of traces that would be deleted based on the
+                max_timestamp_millis, the oldest traces will be deleted first.
             request_ids: A set of request IDs to delete.
 
+            Example usage:
+
+                .. code-block:: python
+
+                    import mlflow
+                    import time
+
+                    client = mlflow.MlflowClient()
+
+                    # Delete all traces in the experiment
+                    client.delete_traces(
+                        experiment_id="0", max_timestamp_millis=time.time_ns() // 1_000_000
+                    )
+
+                    # Delete traces based on max_timestamp_millis and max_traces
+                    # Older traces will be deleted first.
+                    client.delete_traces(
+                        experiment_id="0", max_timestamp_millis=some_timestamp, max_traces=2
+                    )
```

Comment:
```suggestion
                    some_timestamp = ...  # yesterday
                    client.delete_traces(
                        experiment_id="0", max_timestamp_millis=some_timestamp, max_traces=2
                    )
```

can we define `some_timestamp` so we can run this example?